woocommerce/woocommerce
# Hide Product Summary (Product Elements) from Single Product block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





I realized we offer two Product Summary blocks in Single Product block. Since Excerpt variation is used a default Product Summary block in there, I'm hiding Product Summary block from Product Elements.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create new post
2. Insert Single Product block
3. Choose some product
4. Inside Single Product block try inserting Product Summary block
5. **Expected:** There's only one Product Summary block available

Before | After
-- | --
<img width="404" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/697e792b-8a0d-43bd-9e70-d4588faa7598"> | <img width="390" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/72a57694-f2e5-420b-8fdb-199063e64c44">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
